uuv_dave/launch/plot_manipulator.launch

```diff
@@ -0,0 +1,7 @@
+<!-- rqt_plot for force feedback wrist sensor -->
```

Comment:
Start launch files with `<?xml version="1.0"?>` tag.

(Aside - need to check this on existing launch files in the repo.)